test/langtools/tools/javac/patterns/PatternErrorRecovery.java

```diff
@@ -2,7 +2,7 @@
  * @test /nodynamiccopyright/
  * @bug 8268320
  * @summary Verify user-friendly errors are reported for ill-formed pattern.
-* @compile/fail/ref=PatternErrorRecovery.out -XDrawDiagnostics -XDshould-stop.at=FLOW --enable-preview -source ${jdk.version} PatternErrorRecovery.java
+ * @compile/fail/ref=PatternErrorRecovery.out -XDrawDiagnostics -XDshould-stop.at=FLOW --enable-preview -source ${jdk.version} PatternErrorRecovery.java
```

Comment:
why no `@enablePreview` for this test?

Reply:
The test has multiple @compile directives, one of which does not use --enable-preview. (I didn't check closely to see if enablePreview could be toggled on/off for different sets of directives.)

Reply:
You can't toggle. Thanks for the note.